tests/UnitTests/CoreGraphics.drawing/DrawingTest.h

```diff
@@ -0,0 +1,75 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#include <TestFramework.h>
+#include <CoreGraphics/CoreGraphics.h>
+
+#include <Starboard/SmartTypes.h>
+
+namespace testing {
+class DrawTest : public ::testing::Test {
+protected:
+    static woc::unique_cf<CGColorSpaceRef> s_deviceColorSpace;
```

Comment:
nit: would prefer these be accessed similar to parameterized test variables than implicitly available (say getContext